### PR TITLE
Update internlm2 ecosystem

### PR DESCRIPTION
**Pull request by lvhan028**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here and update the documentation.

## Checklist

**Before PR**:

- [ ] Pre-commit or other linting tools are used to fix the potential lint issues.
- [ ] Bug fixes are fully covered by unit tests, the case that causes the bug should be added in the unit tests.
- [ ] The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [ ] The documentation has been modified accordingly, like docstring or example tutorials.

**After PR**:

- [ ] If the modification has potential influence on downstream or other related projects, this PR should be tested with those projects.
- [ ] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

**Comment by QiuZiXian** [comment]
【ollama】 can't import from the merge model:
the merge model is: [quickstart](https://xtuner.readthedocs.io/zh-cn/latest/get_started/quickstart.html) 
I follow [fun-tunning](https://xtuner.readthedocs.io/zh-cn/latest/get_started/quickstart.html)，get the merge model，
then follow [ollama](https://github.com/ollama/ollama/blob/main/docs/modelfile.md#parameter) to convert to gguf
create from mainfest is ok，
but conversion is not.
replay can't stop
image
my template refer[ 1](https://ollama.com/gengshenghong/internlm2/blobs/62fbfd9ed093) :
{{ if .System }}<|im_start|>system
{{ .System }}<|im_end|>
{{ end }}{{ if .Prompt }}<|im_start|>user
{{ .Prompt }}<|im_end|>
{{ end }}<|im_start|>assistant
{{ .Response }}<|im_end|>

not work;
refer [2](https://github.com/InternLM/InternLM/pull/745/files):
echo 'FROM ./internlm2-chat-7b.gguf
TEMPLATE """{{ if .System }}<|im_start|>system
{{ .System }}<|im_end|>
{{ end }}{{ if .Prompt }}<|im_start|>user
{{ .Prompt }}<im_end>
{{ end }}<|im_start|>assistant
{{ .Response }}<|im_end|>"""
PARAMETER stop "<|action_end|>"
PARAMETER stop "<|im_end|>"
not work

File: ecosystem/README.md [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+
+echo "-m
+internlm2-chat-7b.gguf
+--host
+0.0.0.0
+-ngl
+999
+..." > .args
+
+zipalign -j0 \
```
Comment (RunningLeon) [inline ecosystem/README.md:160]: use `llamafile-0.8.6/bin/zipalign` if  `llamafile-0.8.6/bin` not in `PATH`

File: ecosystem/README.md [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+
+SWIFT supports training, inference, evaluation and deployment of LLMs and MLLMs (multimodal large models).
+
+## Inference
+
+### [LMDeploy](https://github.com/InternLM/lmdeploy)
+
+LMDeploy is an efficient toolkit for compressing, deploying, and serving LLMs and VLMs.
+
+With only 4 lines of codes, you can perform `internlm2-chat-7b` inference after `pip install lmdeploy`:
```
Comment (RunningLeon) [inline ecosystem/README.md:37]: `4 lines of code`

File: ecosystem/README.md [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+Regarding the usage of `ollama`, please refer [here](https://github.com/ollama/ollama/tree/main/docs).
+
+### [llamafile](https://github.com/Mozilla-Ocho/llamafile)
+
+llamafile lets you turn large language model (LLM) weights into executables. It combines [llama.cpp](https://github.com/ggerganov/llama.cpp) with [Cosmopolitan Libc](https://github.com/jart/cosmopolitan).
+
+The best practice of deploying InternLM2 using llamafile is shown as below:
+
+- Convert the internlm2 model into GGUF model by `llama.cpp`. Suppose we get `internlm2-chat-7b.gguf` in this step
+- create the llamafile
```
Comment (RunningLeon) [inline ecosystem/README.md:144]: Create

**Review by ZwwWayne — APPROVED** [review]

**Review by RunningLeon — APPROVED** [review]
LGTM

**Pull request merged**